ingest/rules/curate.smk

```diff
@@ -98,6 +97,7 @@ rule curate:
             | ./scripts/parse-gisaid-location \
                 --location-field {params.gisaid_location_field:q} \
                 --strain-field {params.gisaid_strain_field:q} \
+                --annotations {input.location_annotations:q} \
```

Comment:
Ah, this has the downside of potentially getting overridden by geolocation rules downstream. I see you what you mean by needing to document the curation steps for geolocations...